datadog_checks_base/datadog_checks/base/utils/headers.py

```diff
@@ -31,6 +48,7 @@ def headers(agentConfig, **kwargs):
     # Build the request headers
     version = __get_version(agentConfig)
     res = {'User-Agent': 'Datadog Agent/{}'.format(version)}
+    res.update(_get_common_headers())
```

Comment:
I was more meaning you could do 
```suggestion
    res = get_default_headers()
```

Reply:
It's really not that simple, several integrations rely on `headers(agentConfig)`, and `get_default_headers()` ignores any `agentConfig`:

https://github.com/DataDog/integrations-extras/blob/f94cff01855367dae9f81be592a52c36f9e9e8da/logstash/datadog_checks/logstash/logstash.py#L159-L160

https://github.com/DataDog/integrations-extras/blob/f94cff01855367dae9f81be592a52c36f9e9e8da/nextcloud/datadog_checks/nextcloud/nextcloud.py#L52-L53

https://github.com/DataDog/integrations-core/blob/e3d1043a32ea63125448ec67fdaef458a24354d0/couch/datadog_checks/couch/couch.py#L34-L35

I agree there may be room for refactoring, but I'd _really_ like to treat that separately. :-)

Reply:
Not that I know of no, I think `agentConfig` was passed on older Agent versions and in newer ones `datadog_agent` should be used.